python/django/Makefile

```diff
@@ -4,4 +4,4 @@ start:
 
 deps:
 	pip3 install --upgrade setuptools
-	pip3 install django==3.1.4 django-cockroachdb==3.1.2 psycopg2
+	pip3 install django==3.2.* django-cockroachdb==3.2.* psycopg2
```

Comment:
hmm so part of the reason we had this pinned before was so that a update upstream wouldn't break things (https://github.com/cockroachdb/examples-orms/pull/112)

maybe pinning the major/minor version and leaving the patch version open is good enough though. just commenting so you're aware

Reply:
Yes, it should be, and I think it's definitely better than pinning an old, possibly insecure version of Django.

Reply:
I see. I think you're right that explicit pinning is better. We only want these tests to fail due to changes on the CRDB side. I'll make that change.